README.md

```diff
@@ -295,11 +295,20 @@ A list can be used as a queue or stack. `enqueueActionTypes` and `pushActionType
     allIds: [],
   },
   addActionTypes = [],
+  addManyActionTypes = [],
   changeActionTypes = [],
+  changeManyActionTypes = [],
   removeActionTypes = [],
+  removeManyActionTypes = [],
   keyGetter = action => action.payload.id,
   itemGetter = action => ({...action.payload}),
   itemModifier = (item, action) => ({...item, ...action.payload}),
+  addChangeItemsGetter = action => action.payload.reduce((result, item) => ({
+    ...result,
+    [item.id]: item
+  }), {}),
```

Comment:
This is an unnecessary use of immutability: the `accumulator`'s scope inside `Array.prototype.reduce`'s `callback` is well defined. We can safely mutate it. The immutable solution is actually very slow compared to the mutable solution I suggested: https://jsperf.com/mutable-vs-immutable-reduce

I know that redux-data-structures doesn't focus on performance (https://github.com/adrienjt/redux-data-structures#performance), but this is a low hanging fruit.

If you're interested in the details, I suggest running the two snippets with `node --prof`. You'll see that the immutable solution spends most of its time copying arrays:

```
 [Summary]:
   ticks  total  nonlib   name
    104    0.1%    0.1%  JavaScript
  77095   85.7%   85.9%  C++
    242    0.3%    0.3%  GC
    179    0.2%          Shared libraries
  12573   14.0%          Unaccounted

 [C++ entry points]:
   ticks    cpp   total   name
  76917   99.9%   85.5%  T v8::internal::Runtime_CopyDataProperties(int, v8::internal::Object**, v8::internal::Isolate*)
```